src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/fallthrough/InputFallThrough5.java

```diff
@@ -21,7 +21,7 @@ void exhaustiveStatementSane(Object o) {
         switch (o) { // violation below 'Fall .* from the last branch of the switch statement'
             case Object obj, null: ;
         }
-        switch (o) { // violation below 'Fall .* from the last branch of the switch statement'
+        switch (o) {
             case Object obj, null: ; // fall thru
```

Comment:
This looks like a false negative.
https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/b242e7a_2023213303/reports/diff/checkstyle/xref/home/runner/work/checkstyle/checkstyle/.ci-temp/contribution/checkstyle-tester/repositories/checkstyle/src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/fallthrough/InputFallThrough5.java.html#L25

Reply:
Same at https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/b242e7a_2023213303/reports/diff/checkstyle/xref/home/runner/work/checkstyle/checkstyle/.ci-temp/contribution/checkstyle-tester/repositories/checkstyle/src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/fallthrough/InputFallThrough6.java.html#L16

Reply:
@rnveach I am going to open an issue related to this.

as part of the explanation in this pr `fall thru` is not on the last line
https://checkstyle.org/checks/coding/fallthrough.html#FallThrough:~:text=and%20must%20be%20on%20the%20last%20non%2Dempty%20line%20before%20the%20case%20triggering%20the%20warning%20or%20on%20the%20same%20line%20before%20the%20case

Reply:
This is nonsense code, we just not reporting anything on it.
We can create issue issue on this to discuss separately such nonsense structure.
But we need to move forward in conversion of this Check to comments aware.

Reply:
Please confirm if you think separate issue is required.

Reply:
I don't feel comfortable approving this after months of issues with the new implementation and it still producing false results. Users are not gaining anything by merging this sooner except a buggy re-implementation.

> as part of the explanation in this pr fall thru is not on the last line

But it is on the line before the next case.

https://checkstyle.org/checks/coding/fallthrough.html#Description
`The comment containing these words must be all on one line, and must be on the last non-empty line before the case triggering the warning`


https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/b242e7a_2023213303/reports/diff/checkstyle/xref/home/runner/work/checkstyle/checkstyle/.ci-temp/contribution/checkstyle-tester/repositories/checkstyle/src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/fallthrough/InputFallThrough6.java.html#L16
````
14              case 1:
15                   int fallthru = 2; // fallthru
16              case 2: // violation 'Fall through from previous branch of the switch statement'
````
Fall through is the last comment before the case and seems to meet the requirements of the check.

Reply:
I'll relook at the check when I am reassigned. I don't feel comfortable approving this after months of issues with the new implementation if regression is still questionable.